source/bluetooth/MicroBitBLEManager.cpp

```diff
@@ -242,6 +243,7 @@ MicroBitBLEManager::MicroBitBLEManager(MicroBitStorage &_storage) : storage(&_st
  */
 MicroBitBLEManager::MicroBitBLEManager() : storage(NULL)
 {
+    currentMode = MICROBIT_MODE_APPLICATION;
```

Comment:
If this is a bug fix, could you submit it as a different PR?
Same for `inc/bluetooth/MicroBitBLEManager.h`.

Reply:
It is not a bug fix, it is to compensate for the removal of the initializer due to `Error:  #321: data member initializer is not allowed` from the armc5 compiler.